src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const styleAll = sourceString.split(';');
+  const styleTrue = styleAll.filter(style => {
+    return style.split(':')[0].trim().length > 0;
+  });
```

Comment:
Looks like a bit complicated logic here. You can try the approach below. It probably should do the trick as empty string is a falsy expression
```suggestion
  const validStyles = sourceString.split(';').map(style => style.trim()).filter(style => style);
```

Reply:
Yes, I agree, your variant is a bit better